test/fixtures/basic/plugins/error.server.ts

```diff
@@ -0,0 +1,5 @@
+export default defineNuxtPlugin(async () => {
+  if (useRequestHeaders(['x-recurse'])['x-recurse']) {
```

Comment:
Small code style: I would use `x-test-` to make it more readable it is a test suite specific thing

```suggestion
  if (useRequestHeaders(['x-test-recurse-error'])['x-test-recurse-error']) {
```

Reply:
Don't we also have `useRequestHeader` btw?

Reply:
we do, yes.